Detectors/ITSMFT/common/simulation/include/ITSMFTSimulation/Hit.h

```diff
@@ -94,6 +94,9 @@ class Hit : public o2::BasicXYZEHit<Float_t, Float_t>
   Bool_t IsStoppedStart() const { return mTrackStatusStart & kTrackStopped; }
   Bool_t IsAliveStart() const { return mTrackStatusStart & kTrackAlive; }
 
+  // Entrance position setter
+  void SetPosStart(const math_utils::Point3D<Float_t>& p) { mPosStart = p; }
```

Comment:
That sounds fine to me. In these cases, you might consider separating changes to run3 code into different PRs to make it easier to cherry-pick stuff.

Reply:
Do you expect that we need to cherry-pick something for ITS hits? If not I would leave as it is right now, but separate more in the future if it is fine for you.

Reply:
Anchored MC productions are done with the same tag as the reconstruction passes.
Any fix for simulation would be cherry-picked, so potentially, the risk exists.
You can also create a separate commit for that, and we do not squash it upon merging.

Reply:
Makes sense, done.